homeassistant/components/overkiz/config_flow.py

```diff
@@ -28,6 +29,10 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    _config_entry: ConfigEntry | None = None
+    _default_user: None | str = None
+    _default_hub: str = DEFAULT_HUB
```

Comment:
Please initialize these in an `__init__` as they can be different between config entries

Reply:
Addressed in 82def0443f76444ef6886bd3dce574d539176813